### ISSUE TITLE
commonTest should be skipped

### ISSUE THREAD

**Issue by mustafaozhan** [Kotlin/kotlinx-kover#245]
Hello 👋 

Currently kover calculates the coverages for the test classes in `commonTest` in Kotlin Multiplatform projects. It should skip the test folders. If it is not possible we should have option to exclude files by `directory` currently only package option is available.

In Android module `test` folder is already skipped, we should have the same behavior for KMP projects for `commonTest` folder. It causes unfair coverage increase.

**Comment by shanshin** [comment]
Hi,
may you clarify, do you want the Kover report not to take into account the coverage of tests that are located in `commonTest`?

**Comment by shanshin** [comment]
At the moment, there is no way to disable the use of tests from a specific source set. 
As a workaround, you may arrange all tests from the `commonTest` in an individual package and [exclude](https://github.com/Kotlin/kotlinx-kover#configuring-jvm-test-task) its contents from instrumentation.

**Comment by shanshin** [comment]
@mustafaozhan, I would also like to clarify, the classes from `commonTest` were included in the HTML report?

**Comment by mustafaozhan** [comment]
Hello @shanshin thank you for your answers!
> As a workaround, you may arrange all tests from the commonTest in an individual package and [exclude](https://github.com/Kotlin/kotlinx-kover#configuring-jvm-test-task) its contents from instrumentation.

Unfortunately, this will require different setup for each module maybe we need a more general workaround, I tried adding below code into root `build.gradle` but it didn't work, do you have any suggestion ?
```kotlin
koverMerged {
    enable()
    filters {
        classes {
            excludes += listOf(
                "*Test",
                "*.*Test",
                "*.*.*Test",
                "*.*.*.*Test",
                "*.*.*.*.*Test"
            )
        }
    }
}
```

> @mustafaozhan, I would also like to clarify, the classes from commonTest were included in the HTML report?

Yes I can see these classes both in xml and html reports.

**Comment by shanshin** [comment]
This is definitely a bug, classes from test source sets should not get into the report.

**Comment by mustafaozhan** [comment]
@shanshin Thank you for your answer, I agree with you, test sources should be excluded

Also, I think the workaround I have posted should work too, I will create a separate issue for that

**Comment by hoc081098** [comment]
I have the same issue

**Comment by shanshin** [comment]
Fixed in `0.7.0-Alpha`
